tools/prepare-release.js

```diff
@@ -0,0 +1,305 @@
+  else {
+    console.log("- Pre-release PR already based on latest commit, no need to update the branch");
+  }
+
+  console.log();
+  console.log("Commit bumped version to PR branch");
+  if (commitBumpedVersion) {
+    console.log(`- Bump version to ${bumpedVersion}`);
+    const fileResponse = await octokit.repos.getContent({
+      owner, repo, path: `packages/${type}/package.json`
+    });
+
+    const resp = await octokit.repos.createOrUpdateFileContents({
+      owner, repo,
+      branch: prRef,
+      path: `packages/${type}/package.json`,
+      message: `Bump @webref/${type} version from ${version} to ${bumpedVersion}`,
+      content: bumpedPackageFileContents,
+      sha: fileResponse.data.sha,
+    });
+    const bumpedVersionSha = resp.data.commit.sha;
+    console.log(`- Bumped version commit is ${bumpedVersionSha}`);
+
+    if (pendingPR) {
+      // Note: when a pending PR exists, we could in theory reuse the PR branch
+      // directly instead of creating (and then deleting) another one. However,
+      // that would mean force updating that branch to the latest commit of the
+      // main branch before adding a commit with the bumped version. The force
+      // update would effectively close the PR (because the PR would then have
+      // no commit).
+      console.log(`- Force update PR branch ${pendingPR.head.ref} to target ${prRef} branch commit`);
+      await octokit.git.updateRef({
+        owner, repo,
+        ref: `heads/${pendingPR.head.ref}`,
+        sha: bumpedVersionSha, force: true
+      });
+
+      console.log(`- Delete now useless ${prRef} branch`);
+      await octokit.git.deleteRef({
+        owner, repo,
+        ref: `heads/${prRef}`
+      });
+    }
+  }
+  else {
+    console.log("- No need to bump version, PR already up-to-date");
+  }
+
+  console.log();
+  console.log("Create/Update pre-release PR");
+  const title = `📦 Release @webref/${type}@${version}`;
+  const body = `
+**⚠ NEVER add commits to this pull request.**
+
+🤖 This pull request was automatically created to facilitate human review of \`@webref/${type}\` changes.
+
+🧐 Please review the diff below and version numbers. If all looks good, merge this pull request to release the changes to npm.
+
+📦 Latest released \`@webref/${type}\` package was **v${latestReleasedVersion}**. Merging this pull request will release **v${version}**. Make sure that the bump is the right one for the changes.
+
+✍ If any change needs to be made before release, **do not add a commit** to this pull request. Changes should rather be handled in a separate pull request and pushed to the default branch. You may leave this pull request open in the meantime, or close it. The pre-release job will automatically update this pull request or create a new one once the updates have made their way to the default branch.
+
+🛈 Merging this pull request will bump the patch part of the version number in \`packages/${type}/package.json\` so that a new release cycle may begin.
```

Comment:
Maybe explain where the bump happens since it takes some pondering to understand the model?

Reply:
I reformulated the text. I'm not fully convinced that the new text is clear, but when I try to add more text, I have the feeling that it actually becomes more confusing... Happy to take suggestions :)

In the end, what matters is to make it clear that the file changed by the PR is uninteresting for the purpose of issuing a new release. It is just there to reset the cycle after the package has been released.